dot/digest/block_import.go

```diff
@@ -69,10 +60,12 @@ func (h *BlockImportHandler) handleConsensusDigest(d *types.ConsensusDigest, hea
 			return fmt.Errorf("unmarshaling grandpa consensus digest: %w", err)
 		}
 
-		err = h.grandpaState.HandleGRANDPADigest(header, data)
-		if err != nil {
-			return fmt.Errorf("handling grandpa digest: %w", err)
-		}
+		// Shouldn't be needed since the grandpaState is no longer in use here.
+		// err = h.grandpaState.HandleGRANDPADigest(header, data)
+		// if err != nil {
+		// 	return fmt.Errorf("handling grandpa digest: %w", err)
+		// }
+
```

Comment:
you can remove this comment

Reply:
Do you mean I should remove this entire block, or just the comment in line 63 and uncomment the rest?